web/libs/editor/src/stores/types.d.ts

```diff
@@ -29,7 +29,8 @@ type MSTCommentStore = {
 };
 
 type MSTStore = {
-  customButtons: Instance<typeof CustomButton>[];
+  // @todo we can't import CustomButton store here and use it type :(
+  customButtons: any;
```

Comment:
I've just figured out that it works for me 🤯
```suggestion
  customButtons: CustomControlProps.button[];
```

Reply:
that feels a bit weird because it will grab a type from js file which is also based on this definition. I'd leave it like this for now until we find a better way to define MST types